src/App.tsx

```diff
@@ -28,32 +28,30 @@ export default function App() {
         </Container>
 
         {/* DEV */}
-        {import.meta.env.MODE === 'staging' ? (
+        {import.meta.env.MODE === "staging" ? (
           <>
-          <Suspense>
-            {/* for use with dynamically importing dev tools in dev mode, which is not yet set up */}
-            <TanStackRouterDevtools initialIsOpen={false} />
-            <ReactQueryDevtools initialIsOpen={false} />
-          </Suspense>
-          <div className="mt-5 text-danger float-end">
-            API URL: {import.meta.env.VITE_API_URL}
-             - MODE: {import.meta.env.MODE}
-          </div>
+            <Suspense>
+              {/* for use with dynamically importing dev tools in dev mode, which is not yet set up */}
+              <TanStackRouterDevtools initialIsOpen={false} />
+              <ReactQueryDevtools initialIsOpen={false} />
+            </Suspense>
+            <div className="mt-5 text-danger float-end">
+              API URL: {import.meta.env.VITE_API_URL}- MODE:{" "}
+              {import.meta.env.MODE}
+            </div>
           </>
-        ) : null
-        }
+        ) : null}
         {/* END DEV */}
-
       </QueryClientProvider>
     </>
   );
 }
 
 function AppNav({ authUserIsAdmin, authUser }) {
   const { serviceTypes } = Route.useLoaderData();
-  const setAuthUser = useAuthStore((state) => state.setAuthUser);
+  const { setAuthUser } = useAuthStore.getState();
```

Comment:
This was being called in 2 different ways, made it hard to mock. Standardizing on this version.